airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/manifest.yaml

```diff
@@ -0,0 +1,924 @@
+version: 0.65.0
+type: DeclarativeSource
+
+definitions:
+  oauth_authenticator:
+    type: OAuthAuthenticator
+    client_id: "{{ config['credentials']['client_id'] }}"
+    client_secret: "{{ config['credentials']['client_secret'] }}"
+    refresh_token: "{{ config['credentials']['refresh_token'] }}"
+    refresh_request_body: {}
+    token_refresh_endpoint: "{{ 'https://sandbox-lever.auth0.com/oauth/token' if config[ 'environment' ] == 'Sandbox' else 'https://auth.lever.co/oauth/token' }}"
+    grant_type: refresh_token
+
+  basic_authenticator:
+    type: BasicHttpAuthenticator
+    username: "{{ config[''credentials''][''api_key''] }}"
+    password: ""
+
+  selective_authenticator:
+    type: SelectiveAuthenticator
+    authenticator_selection_path: ["credentials", "auth_type"]
+    authenticators:
+      Client: "#/definitions/oauth_authenticator"
+      Api_Key: "#/definitions/basic_authenticator"
```

Comment:
Please change the key to match the existing Api Key`, or add the config migration.

Reply:
Done! I changed it to the original value of `Api Key`